test/app/gh-4235-dont-show-idle-in-fiber-info.result

```diff
@@ -0,0 +1,104 @@
+-- test-run result file version 2
```

Comment:
The test failed one time on my machine:
```
[009] app/gh-4235-dont-show-idle-in-fiber-info.test.>                 [ fail ]
[009] 
[009] Test failed! Result content mismatch:
[009] --- app/gh-4235-dont-show-idle-in-fiber-info.result	Tue Jun  8 21:26:38 2021
[009] +++ var/rejects/app/gh-4235-dont-show-idle-in-fiber-info.reject	Tue Jun  8 21:59:27 2021
[009] @@ -89,11 +89,11 @@
[009]  
[009]  new_idle > ndelta
[009]   | ---
[009] - | - true
[009] + | - false
[009]   | ...
[009]  new_all - new_idle < old_all - old_idle + ndelta
[009]   | ---
[009] - | - true
[009] + | - false
[009]   | ...
[009]  
[009]  conn:close()
```
Couldn't reproduce it again. But still, that makes the test flaky. You might need to design a more robust way to test the patch. Not depending on probabilities, magic constants, timeouts, etc.

Reply:
Thanks for your reply, test is redesigned to be more deterministic.